### DISCUSSION TITLE
hi :) I have some questions

### DISCUSSION THREAD

**Discussion by beckyi** [imzbf/md-editor-rt#148] [Q&A] [answered]
hi :) I'm trying to use editor.

I have some questions to ask you

**1. Can I control the css of the PreviewContent?**

: I have a css file that defines html styles. so I want to apply a custom style to the HTML of the preview area.

**2. How Can I define the width and height of ​​a picture in editor?**

: The format I want is as follows. 
```
//e.g.
![image](https://test.png =width*height)
```

<img width="500" alt="image" src="https://github.com/imzbf/md-editor-rt/assets/18224086/f1152b5d-b7e1-4db5-909b-9a361905fe79">

It seems that the editor is parsing the common mark grammar with the markdown-it module, so it is not reflected.

**3. Can I download the html or access the full html?**

: I want to get default theme or dark theme html

**Comment by imzbf** [comment, marked as answer]
1. Yes, and perhaps this style library is useful for you.

2. For image syntax, we recommend using image tags similar to the GitHub style for display. And there is an extension for it: https://github.com/imzbf/md-editor-extension/tree/develop/packages/rt/components/OriginalImg

3. This operation requires you to define external tags that match the tag class previewed by the editor. There is still a problem with this, which is that mermaid can only be displayed as meta source text

**Reply by beckyi** [reply]
Thanks to answer. 
I don't understand, so I have more questions. 🙏🏻

> Yes, and perhaps this style library is useful for you.

1) There is no prop to pass the css file. Can component receive the css file to be reflected in the preview area?

2) How can I define className or id in Preview's root div element?

<figure>
<img width="2314" alt="image" src="https://github.com/imzbf/md-editor-rt/assets/18224086/45d1b80e-0a71-4215-bd6a-cba9db34fc32">
<figcaption>preview's root div element</figcaption>
</figure>

**Reply by imzbf** [reply]
There is a stylesheet provided for `MdPreview`.

Currently, only the class names for the outer containers are provided.

**Reply by beckyi** [reply]
thanks, I think it was well made with high quality. 👍👍👍
